src/library/scala/collection/immutable/HashMap.scala

```diff
@@ -52,6 +52,9 @@ sealed class HashMap[A, +B] extends AbstractMap[A, B]
   def get(key: A): Option[B] =
     get0(key, computeHash(key), 0)
 
+  override final def contains(key: A): Boolean =
```

Comment:
It looks like `HashSet` is not affected by this change. Do you think it would be worth applying it there too?